robocop/checkers/naming.py

```diff
@@ -653,10 +671,16 @@ def visit_VariableSection(self, node):  # noqa
             if not child.data_tokens:
                 continue
             token = child.data_tokens[0]
-            if token.type == Token.VARIABLE and token.value and not token.value.isupper():
+            if token.type != Token.VARIABLE or not token.value:
+                continue
```

Comment:
It says it is not covered by tests, because youre iterating over every child in variables section and then taking first data token - the first data token for statement in variables section is always Token.Variable so this if condition is always false.

Also, you can check variable directly by using visit_Variable and then getting first data token

Reply:
I refactored it to `visit_Variable`. Is it fine now?